dashboard/src/scenes/organisation/ActionCategoriesSettings.js

```diff
@@ -177,17 +55,17 @@ const ActionCategoriesGroup = ({ groupTitle, categories, onDragAndDrop }) => {
     if (response.ok) {
       refresh();
       setOrganisation(response.data);
-      setIsEditingGroupTitle(false);
       toast.success("Groupe mis à jour. Veuillez notifier vos équipes pour qu'elles rechargent leur app ou leur dashboard");
     } else {
       setOrganisation(oldOrganisation);
     }
   };
 
-  const onDeleteGroup = async () => {
-    if (!window.confirm('Voulez-vous vraiment supprimer ce groupe et toutes ses catégories ? Cette opération est irréversible')) return;
+  const onDeleteGroup = async (groupTitle) => {
     const newActionsGroupedCategories = actionsGroupedCategories.filter((group) => group.groupTitle !== groupTitle);
     const categoriesToDelete = actionsGroupedCategories.find((group) => group.groupTitle === groupTitle).categories;
+    // QUESTION: should we delete the categories from the actions or just remove them from the group?
+    // I forgot the spec
```

Comment:
Ok, it seems that currently we delete them all.

```suggestion
    // QUESTION: should we delete the categories from the actions or just remove them from the group?
    // Currently it's deleted, we should ask for specification.
```